lib/osf-components/addon/components/draft-registration-card/template.hbs

```diff
@@ -67,17 +67,18 @@
             </dl>
             <div class='row'>
                 <div class='col-md-10'>
-                    <OsfLink
+                    <Button
```

Comment:
It's odd to have, one on top of the other, a `<Button><OsfLink>` component and an `<OsfLink><Button>` component. Is this change necessary even with your CSS change? If so, we might want to change the button below this one so that it matches what you're doing here. If just the CSS change works, then I'd revert this change back out.

Reply:
Yea, I definitely got tunnel vision with how to fix that whitespace from showing up, and the CSS change is all that's needed... I'll revert the structure and just keep the CSS changes